scss/civicrm/contact/pages/_contact-merge.scss

```diff
@@ -0,0 +1,44 @@
+/* stylelint-disable max-nesting-depth, selector-max-compound-selectors, selector-no-qualifying-type, selector-max-id */
+
+#{civi-page('contact-merge')} {
+  .crm-container {
+    .crm-contact-merge-form-block {
+      padding-top: 20px;
+
+      .help,
+      .message.status {
+        margin: $crm-status-bar-no-top;
```

Comment:
i'm sorry to keep bringing this up every time, but the fact that we have to re-define over and over the margins for these common elements really doesn't smell right to me. Is it really possible that we can't have just *one* (or two, at best) margin value across civicrm?

Reply:
We discussed this sometime ago though, based on it's parents we are not sure with multiple scenarios on where we apply what so, we came up with variables like `$crm-status-bar-no-top` and `$crm-status-bar-no-bottom`